IanGRap/Squish
# Migrate to Unity 2017

Contributors: Austin, Ian, Alfredo

Goal:
- Fix the controller bugs by moving to Unity 2017

What changed:
- Literally everything

How to test:
- Load up this branch in unity version 2017.4.21f1
- Make sure everything works
- Make a build and make sure everything works
